src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -115,32 +115,47 @@ class Artwork extends React.Component<Props, any> {
     }
   }
 
-  saleMessage() {
-    const artwork = this.props.artwork
-    if (artwork.is_in_auction && artwork.sale_artwork) {
-      if (!artwork.sale_artwork.sale.is_closed) {
-        let numberOfBids = null
-        try {
-          numberOfBids = artwork.sale_artwork.bidder_positions_count
-        } catch (e) {
-          console.error(`Sentry issue #274707594 triggered with props: ${JSON.stringify(this.props)}`)
-          return null
-        }
-        let text = artwork.sale_artwork.opening_bid.display
-        if (numberOfBids > 0) {
-          text = `${artwork.sale_artwork.current_bid.display} (${numberOfBids} bid${numberOfBids === 1 ? "" : "s"})`
-        }
-        return (
-          <View style={{ flexDirection: "row" }}>
-            <Image style={{ marginRight: 4 }} source={require("../../../../images/paddle.png")} />
-            <SerifText style={styles.text}>{text}</SerifText>
-          </View>
-        )
-      } else {
-        return <SerifText style={styles.text}>Auction Closed</SerifText>
-      }
-    } else {
-      return artwork.sale_message && <SerifText style={styles.text}>{artwork.sale_message}</SerifText>
+  saleInfoLine() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inClosedAuction = sale && sale.is_auction && sale.is_closed
+
+    return (
+      <Flex flexDirection={"row"} mb={2}>
+        <Sans color={color("black100")} weight={"medium"} size={"2"}>
+          {inClosedAuction ? "Bidding closed" : this.saleMessageOrBidInfo()}{" "}
+        </Sans>
+        <Sans size={"2"} color={color("black60")}>
+          {!inClosedAuction && this.auctionInfo()}
+        </Sans>
+      </Flex>
+    )
+  }
+
+  saleMessageOrBidInfo() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inRunningAuction = sale && sale.is_auction && !sale.is_closed
+
+    if (inRunningAuction) {
+      const sa = artwork.sale_artwork
+      return sa.current_bid.display
```

Comment:
Can you add an additional guard here around `current_bid && current_bid.display`? I've noticed that occasionally this will crash out (not sure why).